### DISCUSSION TITLE
Chia farm 300TB VS 1PB - What’s wrong?

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#10961] [Support]
Finally, I got my 1PB of plots. But I do have a problem with that. The more I plot the less blocks I earn. It does not affect my earning on the pool however, daily earnings are increasing at this point.
85% of all block rewards I received with 200-500TB, and the remaining 15% was around 800-1000TB. The network at those times was always around 30EB. So, with only 400TB I received a block every 2-3 weeks, but with 1PB I got none for 45 days.
Can anyone please explain what’s wrong here? I have attached my Debug here: [debug.log](https://github.com/Chia-Network/chia-blockchain/files/8382346/debug.log)

**1.	Chia installed on Ubuntu and only farming on this machine.
2.	NO errors in logs
3.	8444 port is open. I even add 300 peers now to test.
4.	Farming only one more fork - Silicoin
5. 	All hard drives are connected directly to the farmer via USB C-type 10Gb. I use 5 ports, each has it's own controller.
6.	! I do have some external Wallets connected to my node however, not sure if that’s ok.**

![rig](https://user-images.githubusercontent.com/87495802/160885783-02bd5cfe-61d8-4560-ae97-63828e022301.jpg)
![Screenshot_10](https://user-images.githubusercontent.com/87495802/160875784-a5c35801-c1aa-4bca-9428-c636b7e6cc99.png)

**Comment by loppefaaret** [comment]
there is the potential for ups a downs - although it is not common, reports of up to 10x swings have happened... a 5x swing for 1PiB is about a month, have you had 0 blocks the last 30 days with 1 PiB ? if yes, that means you are about x5 bad luck, which is going to happen to alot of people i'd imagine at some point, myself included, more than once.

**Reply by ghost** [reply]
I have 0 blocks for the past 47 days. I feel like i need to split all drives on two systems to see if anything will change.

**Comment by isipisi89** [comment]
How much energy dose your farm consume with 65 Harddrives? 650 Watt?

**Reply by ghost** [reply]
i haven't measure any power consumption from the wall, but each enclosure rated 60W,

**Comment by ghost** [comment]
I have changed configurations a bit. Now this whole wall is acting as Harvester only, and actual node is on another machine with i9 10980XE overclocked to 5GH on all cores. Will run this configuration for a week to see if anything will change in terms of rewards. Still farming on the pool.

**Using 1 Gigabit ethernet switch.**  Unfortunately don't have 10 Gigabit switch. 

![Screenshot_1](https://user-images.githubusercontent.com/87495802/161886780-d8385b7e-b5f2-48af-9ac9-9ea891abc064.png)
![Screenshot_2](https://user-images.githubusercontent.com/87495802/161887137-ecda8245-0e67-4ccb-bb5c-8a74ddff8b6d.png)

**Comment by ghost** [comment]
Harvesting over local network to the farmer is a failure, i think. I have noticed ~10% drop in revenue while pooling. Feels like it's better when all drives are connected directly to the farmer via cables.
